spring/spring/pom.xml

```diff
@@ -183,6 +183,12 @@
             <artifactId>commons-dbcp</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.aspectj</groupId>
+            <artifactId>aspectjweaver</artifactId>
+            <version>${version.aspectj.aspectjweaver}</version>
```

Comment:
Since both Spring modules use the same version I would recommend to put it in `parent/pom.xml` in DependencyManagement section.